querybook/server/lib/ai_assistant/base_ai_assistant.py

```diff
@@ -254,8 +254,25 @@ def generate_sql_query(
             session=session,
         )
 
+        self._generate_sql_query(
+            language=query_engine.language,
+            question=question,
+            table_schemas=table_schemas,
+            original_query=original_query,
+            socket=socket,
+        )
+
+    def _generate_sql_query(
+        self,
+        language: str,
+        question: str,
+        table_schemas: list[str],
+        original_query: str = None,
+        socket=None,
```

Comment:
is socket not required here? how come the autofix one requires it

Reply:
yes, it is required actually. let me remove the default value.